### PR TITLE
create media player

### PR DESCRIPTION
**Pull request by victorbaxter**
## [Pull request title]
create media player
### [Redmine link]
https://edu-redmine.framgia.vn/issues/7648

### CODE REVIEW

**Comment by victorbaxter** [comment]
ready

File: app/src/main/java/phuchh/com/music_60/mediaplayer/MediaPlayerManager.java [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+            mCurrentIndex = getTracks().size() - CHANGE_POSITION;
+        else
+            mCurrentIndex = --mCurrentIndex;
+        create(mCurrentIndex);
+        mListener.onChangeTrack();
+    }
+
+    @Override
+    public void onCompletion(MediaPlayer mediaPlayer) {
+        if (mCurrentIndex == mTracks.size() - CHANGE_POSITION && mState != PlayerStatus.STOPPED)
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/phuchh/com/music_60/mediaplayer/MediaPlayerManager.java]: if else should add {}

File: app/src/main/java/phuchh/com/music_60/mediaplayer/MediaPlayerManager.java [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        if (mMediaPlayer != null) {
+            mMediaPlayer.stop();
+            mState = PlayerStatus.STOPPED;
+            mListener.onTrackStopped();
+        }
+    }
+
+    @Override
+    public int getDuration() {
+        if (mMediaPlayer != null && mState >= PlayerStatus.STARTED)
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/phuchh/com/music_60/mediaplayer/MediaPlayerManager.java]: if else should add {}

File: app/src/main/java/phuchh/com/music_60/mediaplayer/MediaPlayerManager.java [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+            return mMediaPlayer.getDuration();
+        return 0;
+    }
+
+    @Override
+    public int getCurrentPosition() {
+        if (mMediaPlayer != null && mState >= PlayerStatus.STARTED)
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/phuchh/com/music_60/mediaplayer/MediaPlayerManager.java]: if else should add {}

File: app/src/main/java/phuchh/com/music_60/mediaplayer/MediaPlayerManager.java [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+    public void next() {
+        mCurrentIndex++;
+        mCurrentIndex = mCurrentIndex == mTracks.size() ? 0 : mCurrentIndex;
+        create(mCurrentIndex);
+        mListener.onChangeTrack();
+    }
+
+    @Override
+    public void previous() {
+        if (mCurrentIndex == 0)
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/phuchh/com/music_60/mediaplayer/MediaPlayerManager.java]: if else should add {}

**Comment by victorbaxter** [comment]
ready

**Pull request merged**